app/controllers/invoices_controller.rb

```diff
@@ -1,7 +1,151 @@
 class InvoicesController < ApplicationController
+  PATH = 'app/assets/stylesheets'
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @invoices = current_company.invoices
+  end
 
   def new
     @invoice = Invoice.new
     @invoice.line_items << LineItem.new
   end
+
+  def create
+    @invoice = Invoice.new(invoice_params)
+    @invoice.date_of_issue = Date.strptime(params[:invoice][:date_of_issue],"%m/%d/%Y")
```

Comment:
Move parsing logic to a common method in ApplicationController. Use I18n helpers to parse the time instead of using hardcoded date formats.